internal-export-file/export-report-pdf/src/export-report-pdf.py

```diff
@@ -193,13 +193,19 @@ def _process_list(
 
         if entities_list is not None:
             list_marking = None
-            if file_markings:
-                list_marking = file_markings[-1]["definition"]
+            if len(file_markings) != 0:
```

Comment:
Are we sure file_markings is always a defined array ?

Reply:
Good question !
From the client, we send an array
And I tried from the graphql playground, if I send null or undefined ( or anything else that isn't an array ) it throws me an error